synthetics/test_schema.go

```diff
@@ -9,7 +9,7 @@ import (
 
 func makeTestSchema(mode schemaMode) map[string]*schema.Schema {
 	return map[string]*schema.Schema{
-		"id": {
```

Comment:
I do not get this change, could you explain? I read it as a map with strings as keys and pointer to `schema.Schema` as values. Before it was clear to me that this field represents id, now I'm wondering where `idKey` variable is defined. Dunno if I make sense I might be missing something. Please explain :)

\+ whole definition looked coherent before, when each key was defined as string literal.

Reply:
It is defined in:
```shell
➜  terraform-provider-kentik-synthetics git:(master) rg -i 'idKey ='
synthetics/schema.go
8:	idKey = "id"
```

ID is used in multiple places - more that other properties. That's why I extracted a const for it. Creating const for every key is not worth it IMO. All keys are used in `synthetics/test_schema.go` and `synthetics/test_conversions.go`. ID key is used in following files:
```
synthetics/test_conversions.go
synthetics/data_source_agent.go
synthetics/agent_schema.go
synthetics/resource_test_impl.go
synthetics/data_source_test_impl.go
```